cloud/awsprovider.go

```diff
@@ -546,15 +572,23 @@ func (aws *AWS) DownloadPricingData() error {
 						if ok {
 							aws.Pricing[key].OnDemand = offerTerm
 							aws.Pricing[spotKey].OnDemand = offerTerm
-							if strings.HasSuffix(key, "EBS:VolumeP-IOPS.piops") {
+							if strings.Contains(key, "EBS:VolumeP-IOPS.piops") {
 								// If the specific UsageType is the per IO cost used on io1 volumes
 								// we need to add the per IO cost to the io1 PV cost
 								curr_region := strings.Split(key, ",")[0]
 								cost := offerTerm.PriceDimensions[sku.(string)+OnDemandRateCode+HourlyRateCode].PricePerUnit.USD
-								cost_F, _ := strconv.ParseFloat(cost, 64)
-								piops_key := curr_region + ",EBS:VolumeUsage.piops"
+								// UsageType in most regions starts with a region identifier
+								// Here we get that region identifier and then use it as part of
+								// the key for looking up and storing the per IO cost with the io1 price
+								billing_region_code := regionToBillingRegionCode[curr_region]
+								piops_key := ""
+								if billing_region_code != "" {
+									piops_key = curr_region + "," + billing_region_code + "-EBS:VolumeUsage.piops"
```

Comment:
nit: Go convention is to use camelCase vs underscores-- linters complain about this in general.

Reply:
(ditto for other underscores)

Reply:
fixed these - I noticed I also did this in my previous Glue PR, so I'll submit a change for that as well.